components/profileSection/profileSection.tsx

```diff
@@ -13,61 +14,174 @@ import WorkDetails from './workDetails';
 import RecentEvent from './recentEvent';
 import AIBlock from './aiBlock';
 import { useStores } from '@/stores';
+import { capitalizeString } from '@/helpers/common';
+
+interface ContactInfo {
+  label: string;
+  value: string;
+}
 
 export default function ProfileSection() {
   const { ticketStore } = useStores();
   const { ticketDetails } = ticketStore;
   const { contact } = ticketDetails || {};
+  const [contactInfo, setContactInfo] = useState<ContactInfo[]>([]);
+
+  const formatTime = (isoString: string) => {
```

Comment:
move this function to common files, since it's also used in tacking events

Reply:
Done